pkg/controller/component/component_controller.go

```diff
@@ -2,13 +2,16 @@ package component
 
 import (
 	"context"
+	e "errors"
 	"fmt"
-	"github.com/openshift/api/apps/v1"
 
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
+	"strconv"
```

Comment:
Can we put strconv with the other standard libraries to keep our imports organized? Ref: https://github.com/golang/go/wiki/CodeReviewComments#imports